src/style.css

```diff
@@ -0,0 +1,57 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.mojo-logo-link {
+  height: 40px;
+  width: 40px;
```

Comment:
FYI: Try to use only height or width to best sizing an image.

Reply:
fixed